component/inter_request_state.cc

```diff
@@ -23,9 +24,17 @@ void Self::CreateForBrowserContext(content::BrowserContext* c, PrefService* p) {
   DCHECK(p);
   auto owned = std::make_unique<ipfs::InterRequestState>(c->GetPath(), p);
   c->SetUserData(user_data_key, std::move(owned));
+  auto* cpsp = content::ChildProcessSecurityPolicy::GetInstance();
+  cpsp->RegisterWebSafeScheme("ipfs");
```

Comment:
Feel free to ignore the chromium_edits/ , as usual. This is the essence of the change.